schema.graphql

```diff
@@ -51,187 +118,650 @@ enum Status {
+
+  """
+  Total number of all Spaces which Account is following
+  """
   followingSpacesCount: Int!
 
+  """
+  Many-To-Many relationship between Account and done Activity in network through NewsFeed (foreign key - "account").
+  Each Activity has "event<EventName>" and "post" fields which can be used for adding created Posts to user's Feed.
+  """
   feeds: [NewsFeed]! @derivedFrom(field: "account")
+
+  """
+  Many-To-Many relationship between Account and done Activity in network through Notification (foreign key - "account").
+  """
   notifications: [Notification]! @derivedFrom(field: "account")
+
+  """
+  One-To-Many relationship with Activity which are done by Account (foreign key - "account")
+  """
   activities: [Activity]! @derivedFrom(field: "account")
+
+  """
+  One-To-Many relationship with Reaction which are done by Account (foreign key - "account")
+  """
   reactions: [Reaction]! @derivedFrom(field: "account")
 
+  """
+  DateTime when Account has been updated by Profiles.ProfileUpdated event
+  """
   updatedAtTime: DateTime
+
+  """
+  Block height when Account has been updated by Profiles.ProfileUpdated event
+  """
   updatedAtBlock: BigInt
 }
 
+"""
+Post entity
+"""
 type Post @entity {
+  """
+  Post ID. The same as in blockchain.
+  """
   id: ID!
-  parentPost: Post
+
+  """
+  One-to-One relationship with Post. This field has value only if current Post is a Comment or Reply to Comment and contains relation with top level Regular Post.
+  """
   rootPost: Post
+
+  """
+  One-to-One relationship with Post. This field has value only if current Post is a Reply to Comment and contains relation with Comment Post or another Reply (in case there is discussion within context of some Comment).
+  """
+  parentPost: Post
+
+  """
+  One-to-One relationship with Post which has been shared. Current Post is a new Post which has been created during sharing action and can contain additional body as a comment to sharing Post. "sharedPost" is relation with Post which has been shared
+  """
   sharedPost: Post
+
+  """
+  Is current Post a Comment to Regular Post or Comment Post
+  """
   isComment: Boolean! @index
+
+  """
+  Is current post hidden
+  """
   hidden: Boolean! @index
 
+  """
+  One-To-One relationship with Post's owner Account entity. Currently we don't have Post transfer functionality.
+  """
   ownedByAccount: Account!
+
+  """
+  One-To-One relationship with Post's creator Account entity
+  """
   createdByAccount: Account!
+
+  """
+  Block height when Post has been created
+  """
   createdAtBlock: BigInt
+
+  """
+  DateTime when Post has been created
+  """
   createdAtTime: DateTime @index
+
+  """
+  Day when Post has been created
+  """
   createdOnDay: DateTime
+
+  """
+  Time when Post has been created
+  """
   updatedAtTime: DateTime
+
+  """
+  One-To-One relationship with Space current Post has been created on. Can be null in case Post is deleted (moved to Space with ID === null)
```

Comment:
post cannot be deleted at all. It can be moved to Space with ID === null (just set space field to null) and it means that it's deleted. But user can restore deleted post by moving it to some live Space.